packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -217,3 +227,36 @@ export async function promptToCreateProjectIfNotExistsAsync(
     privacy,
   });
 }
+
+export function isExpoUpdatesInstalled(projectDir: string): boolean {
+  const packageJson = getPackageJson(projectDir);
+  return !!(packageJson.dependencies && 'expo-updates' in packageJson.dependencies);
+}
+
+export function isExpoUpdatesInstalledOrAvailable(
+  projectDir: string,
+  sdkVersion?: string
+): boolean {
+  // before sdk 44, expo-updates was included in with the expo module
+  if (sdkVersion && semver.lt(sdkVersion, '44.0.0')) {
+    return true;
+  }
+
+  return isExpoUpdatesInstalled(projectDir);
+}
+
+export async function installExpoUpdatesAsync(
+  projectDir: string,
+  { nonInteractive }: { nonInteractive: boolean }
+): Promise<void> {
+  Log.newLine();
+  Log.log(`Running ${chalk.bold('expo install expo-updates')}`);
+  Log.newLine();
+  await expoCommandAsync(projectDir, ['install', 'expo-updates']);
+  Log.newLine();
+  if (await getVcsClient().isCommitRequiredAsync()) {
+    await reviewAndCommitChangesAsync('Install expo-updates', {
```

Comment:
do not import from builds here, make this function only install the package (without all that gist stuff) and in builds directory create a wrapper that calls this function and commits the changes after that